tests/testsuite/vendor.rs

```diff
@@ -1254,6 +1284,7 @@ fn vendor_crate_with_ws_inherit() {
                     [package]
                     name = "foo"
                     version = "0.1.0"
+                    edition = "2021"
```

Comment:
Add a default edition to prevent `[WARNING] no edition set: defaulting to the 2015 edition while the latest is 2021`.